frontend/src/pages/OnboardingPageV2/OnboardingPageV2.styles.scss

```diff
@@ -0,0 +1,233 @@
+.setup-flow {
+	margin-top: 74px;
+	z-index: 1;
+
+	&__header {
+		background: rgba(11, 12, 14, 0.7);
+		backdrop-filter: blur(20px);
+		padding: 8px 16px;
+
+		&--sticky {
+			padding-top: 16px;
+		}
+	}
+
+	&__question-block {
+		position: relative;
+		padding-left: 64px;
+		margin-bottom: 40px;
+		color: #e0e0e0;
+		background-color: #0b0c0e;
+
+		&:before {
+			content: '';
+			position: absolute;
+			left: -5px;
+			top: -5px;
+			width: 30px;
+			height: 30px;
+			border-radius: 50%;
+			background-color: var(--Ink-400, #121317);
+			border: 1px solid var(--Greyscale-Slate-400, #1d212d);
+			color: #1e1e1e;
+			display: flex;
+			align-items: center;
+			justify-content: center;
+			font-weight: bold;
+		}
+
+		&:not(:last-child):after {
+			content: '';
+			position: absolute;
+			left: 10px;
+			top: 30px;
+			bottom: -40px;
+			width: 1px;
+			background: linear-gradient(
+				to bottom,
+				#1d212d 0%,
+				#1d212d 50%,
+				transparent 50%,
+				transparent 100%
+			);
+			background-size: 2px 10px;
+		}
+	}
+
+	&__heading {
+		font-size: 24px;
+		margin-bottom: 12px;
+		color: #fff;
+		font-weight: 600;
+	}
+
+	&__description {
+		font-size: 16px;
+		color: var(--Vanilla-400, #c0c1c3);
+		margin-bottom: 48px !important;
+		line-height: 1.5;
+	}
+
+	&__search {
+		margin-bottom: 24px !important;
+	}
+
+	&__radio-buttons {
+		// display: flex;
+		// flex-wrap: wrap;
+		// gap: 12px;
+	}
+
+	&__radio-label {
+		display: flex;
+		cursor: pointer;
+		white-space: nowrap;
+	}
+
+	&__radio-input {
+		display: none;
+	}
+
+	&__radio-custom {
+		padding: 10px 20px;
+		background-color: #2a2a2a;
+		color: var(--Vanilla-400, #c0c1c3);
+		font-size: 16px;
+		font-style: normal;
+		font-weight: 400;
+		line-height: 22px; /* 137.5% */
+		transition: all 0.3s ease;
+		border-radius: 4px;
+		background: var(--Ink-400, #121317);
+		cursor: pointer;
+		box-shadow: 0px 4px 4px 0px rgba(0, 0, 0, 0.19);
+
+		&:hover {
+			background: var(--Robin-500, #4e74f8);
+			box-shadow: 0 4px 8px rgba(58, 134, 255, 0.2);
+		}
+
+		&__text {
+			color: var(--Vanilla-400, #c0c1c3);
```

Comment:
incorrect variable names

Reply:
There are few changes need to made post this feature is done with the functionality. I will fix this once the flow is functional.